### DISCUSSION TITLE
How to add attachments

### DISCUSSION THREAD

**Discussion by 867810770** [TriliumNext/Trilium#3145] [Q&A]
How to add attachments to notes?

**Comment by XXXJetfireXXX** [comment]
You mean like images and such? The easiest way to do so is (from my perspective) by using `Ctrl+C`-`Ctrl+V`: just copy the image you want to clipboard and paste it in any Trillium note :)

**Comment by zadam** [comment]
Alternatively, you can use "Import files" dialog:

![image](https://user-images.githubusercontent.com/617641/191113517-b4e6c283-1bb0-4150-9cfc-76bfa082b561.png)
